pick_position_x: 600.0
pick_position_y: 200.0
pick_position_z: 100.0
pick_orientation_x: 0.0
pick_orientation_y: 0.707
pick_orientation_z: 0.707
pick_orientation_w: 0.0  
pick_close_angle: 30.0 

place_position_x: 600.0
place_position_y: 300.0
place_position_z: 100.0
place_orientation_x: 0.0
place_orientation_y: 0.707
place_orientation_z: 0.707
place_orientation_w: 0.0  
place_open_angle: 20.0

bouncelessF2E_setup_position_x: 600.0
bouncelessF2E_setup_position_y: 300.0
bouncelessF2E_setup_position_z: 200.0
bouncelessF2E_setup_orientation_x: 0.0
bouncelessF2E_setup_orientation_y: 0.707
bouncelessF2E_setup_orientation_z: 0.707
bouncelessF2E_setup_orientation_w: 0.0

pushTranslate_safe_position_x: 600.0
pushTranslate_safe_position_y: 200.0
pushTranslate_safe_position_z: 200.0
pushTranslate_safe_orientation_x: 0.0
pushTranslate_safe_orientation_y: 0.707
pushTranslate_safe_orientation_z: 0.707
pushTranslate_safe_orientation_w: 0.0
pushTranslate_push_stop_position_x: 600.0
pushTranslate_push_stop_position_y: 200.0
pushTranslate_push_stop_position_z: 72.0
pushTranslate_push_stop_orientation_x: 0.0
pushTranslate_push_stop_orientation_y: 0.707
pushTranslate_push_stop_orientation_z: 0.707
pushTranslate_push_stop_orientation_w: 0.0

vibrate_angle: 43.0

topple_sleep: 83000
topple_pose_position_x: 600.0
topple_pose_position_y: 700.0
topple_pose_position_z: 800.0
topple_pose_orientation_x: 0.707
topple_pose_orientation_y: 0.0
topple_pose_orientation_z: 0.0
topple_pose_orientation_w: -0.707

rotateInFingers_zone: 1
rotateInFingers_push_position_x: 0.0
rotateInFingers_push_position_y: 0.0
rotateInFingers_push_position_z: 0.0 
rotateInFingers_push_orientation_x: 0.0
rotateInFingers_push_orientation_y: 0.707
rotateInFingers_push_orientation_z: 0.707
rotateInFingers_push_orientation_w: 0.0

rollInHand_roll_angle: 10.0
rollInHand_safe_position_x: 700.0
rollInHand_safe_position_y: 325.0
rollInHand_safe_position_z: 810.0 
rollInHand_safe_orientation_x: 1.0
rollInHand_safe_orientation_y: 0.0
rollInHand_safe_orientation_z: 0.0
rollInHand_safe_orientation_w: 0.0
rollInHand_push_position_x: 700.0
rollInHand_push_position_y: 325.0
rollInHand_push_position_z: 810.0 
rollInHand_push_orientation_x: 1.0
rollInHand_push_orientation_y: 0.0
rollInHand_push_orientation_z: 0.0
rollInHand_push_orientation_w: 0.0
rollInHand_push_intm_position_x: 700.0
rollInHand_push_intm_position_y: 325.0
rollInHand_push_intm_position_z: 810.0 
rollInHand_push_intm_orientation_x: 0.0
rollInHand_push_intm_orientation_y: 0.707
rollInHand_push_intm_orientation_z: 0.0
rollInHand_push_intm_orientation_w: 0.707
rollInHand_push_inv_position_x: 700.0
rollInHand_push_inv_position_y: 325.0
rollInHand_push_inv_position_z: 810.0 
rollInHand_push_inv_orientation_x: 0.0
rollInHand_push_inv_orientation_y: 0.0
rollInHand_push_inv_orientation_z: 0.0
rollInHand_push_inv_orientation_w: 1.0

groundRoll_tcp: 40.0
groundRoll_ori: 20.0
groundRoll_roll_position_x: 650.0
groundRoll_roll_position_y: 200.0
groundRoll_roll_position_z: 67.5 
groundRoll_roll_orientation_x: 0.0
groundRoll_roll_orientation_y: 0.707
groundRoll_roll_orientation_z: 0.707
groundRoll_roll_orientation_w: 0.0
groundRoll_roll_place_position_x: 705.0
groundRoll_roll_place_position_y: 200.0
groundRoll_roll_place_position_z: 67.5 
groundRoll_roll_place_orientation_x: 0.0
groundRoll_roll_place_orientation_y: 0.707
groundRoll_roll_place_orientation_z: 0.707
groundRoll_roll_place_orientation_w: 0.0

flipInHand_sleep: 670000

flipInAir_sleep: 570000
flipInAir_flip_position_x: 612.0
flipInAir_flip_position_y: 188.0
flipInAir_flip_position_z: 132.0 
flipInAir_flip_orientation_x: 0.487
flipInAir_flip_orientation_y: 0.720
flipInAir_flip_orientation_z: 0.410
flipInAir_flip_orientation_w: -0.275

vertCyl_tcp: 20.0
vertCyl_ori: 10.0
vertCyl_pick_position_x: 600.0
vertCyl_pick_position_y: 235.0
vertCyl_pick_position_z: 79.5 
vertCyl_pick_orientation_x: 0.0
vertCyl_pick_orientation_y: 0.707
vertCyl_pick_orientation_z: 0.707
vertCyl_pick_orientation_w: 0.0
vertCyl_pick2_position_x: 600.0
vertCyl_pick2_position_y: 235.0
vertCyl_pick2_position_z: 194.0 
vertCyl_pick2_orientation_x: 0.0
vertCyl_pick2_orientation_y: 0.707
vertCyl_pick2_orientation_z: 0.707
vertCyl_pick2_orientation_w: 0.0
vertCyl_place_position_x: 600.0
vertCyl_place_position_y: 200.0
vertCyl_place_position_z: 78.5 
vertCyl_place_orientation_x: 0.0
vertCyl_place_orientation_y: 0.707
vertCyl_place_orientation_z: 0.707
vertCyl_place_orientation_w: 0.0
vertCyl_place2_position_x: 593.0
vertCyl_place2_position_y: 207.0
vertCyl_place2_position_z: 58.5
vertCyl_place2_orientation_x: 0.0
vertCyl_place2_orientation_y: 0.707
vertCyl_place2_orientation_z: 0.707
vertCyl_place2_orientation_w: 0.0

e2f_speed: 0.9
e2f_fspeed: 0.78
e2f_sleep: 500
e2f_partial_open_angle: 15.0
e2f_pose_position.x: 593.0
e2f_pose_position.y: 207.0
e2f_pose_position.z: 58.5 
e2f_pose_orientation.x: 0.0
e2f_pose_orientation.y: 0.707
e2f_pose_orientation.z: 0.707
e2f_pose_orientation.w: 0.0
e2f_pose1_position.x: 593.0
e2f_pose1_position.y: 207.0
e2f_pose1_position.z: 58.5 
e2f_pose1_orientation.x: 0.0
e2f_pose1_orientation.y: 0.707
e2f_pose1_orientation.z: 0.707
e2f_pose1_orientation.w: 0.0
e2f_pose2_position.x: 593.0
e2f_pose2_position.y: 207.0
e2f_pose2_position.z: 58.5 
e2f_pose2_orientation.x: 0.0
e2f_pose2_orientation.y: 0.707
e2f_pose2_orientation.z: 0.707
e2f_pose2_orientation.w: 0.0
